pkg/cfaws/assumer_aws_iam.go

```diff
@@ -41,17 +50,25 @@ func (aia *AwsIamAssumer) AssumeTerminal(ctx context.Context, c *Profile, config
 		}),
 	}
 
-	//load the creds from the credentials file
+	// load the creds from the credentials file
 	cfg, err := config.LoadDefaultConfig(ctx, opts...)
 	if err != nil {
 		return aws.Credentials{}, err
 	}
 
-	creds, err := aws.NewCredentialsCache(cfg.Credentials).Retrieve(ctx)
+	credentials, err = aws.NewCredentialsCache(cfg.Credentials).Retrieve(ctx)
 	if err != nil {
 		return aws.Credentials{}, err
 	}
-	return creds, nil
+
+	//inform the user about using the secure storage to securely store IAM user credentials
+
+	fmt.Fprintf(color.Error, "Profile %s has plaintext credentials stored in the AWS credentials file.\n", c.Name)
+	fmt.Fprintf(color.Error, "With Granted you can store these credentials in secure storage.\n")
+	fmt.Fprintf(color.Error, "To move the credentials to secure storage, run `granted credentials import %s`.\n", c.Name)
```

Comment:
These prompts don't appear for me in my testing, @JoshuaWilkes have you seen them in your local dev environments?

Reply:
Fixed in latest changes